src/interface/polar.jl

```diff
@@ -49,6 +49,15 @@ struct PolarViaSVD{SVDAlg} <: AbstractAlgorithm
     svdalg::SVDAlg
 end
 
+"""
```

Comment:
I think the other algorithms are all defined in the `src/interface/decompositions.jl` file. How do you feel about also moving these (including the `PolarViaSVD` above) there?